lib/Optimizer/Transforms/CMakeLists.txt

```diff
@@ -17,6 +17,8 @@ add_cudaq_library(OptTransforms
   ApplyOpSpecialization.cpp
   ArgumentSynthesis.cpp
   BasisConversion.cpp
+  CollapseStores.cpp
+  CollapseStoresPatterns.cpp
```

Comment:
It looks like these ought to be combined into 1 file. We put the patterns in the same source file with the pass.

Reply:
Renamed to WriteAfterWriteEliminationcpp and WriteAfterWriteEliminationPatterns.inc (used in loop unroll as well)